.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/andrew/firedrake/bin/python3"
+}
```

Comment:
Please don't commit editor-specific files. You can fix this by doing `git rm` of this file and committing that deletion (and then squashing with 992002d).